docs/proposals/dataset_and_simulation.md

```diff
@@ -0,0 +1,149 @@
+The specific research results of AI Benchmark for cloud-side-device collaboration can be found in this [document(in chinese)](https://github.com/iszhyang/AI-Benchmark-for-Cloud-Edge-Device).
+
+As a very promising community in edge AI Benchmarking, ianvs still has the following problems to be solved
+
+- The automatic dataset download and management features are missing, and users still need tedious operations to use the dataset.
+- Repeated deployment of the edge-cloud collaboration environment is too heavy. The distributed collaborative system simulation feature of ianvs needs to be developed urgently.
+
+This proposal provides features in datasets, edge-cloud collaborative AI system simulation, etc. for the current version of Ianvs
+
+## Goals
+
+The project is committed to building an edge-cloud collaborative AI Bechmark for edge-cloud collaborative lifelong detection in the Kubeedge open source community.
+
+- Provide downloadable open source datasets and data processing scripts for the difficulty in obtaining datasets and supporting algorithms
+- Provides industrial-grade distributed collaborative system simulation for the problem that the repeated deployment of edge-cloud collaboration is too heavy
+- In view of the high cost of solution selection and the obscure problem of value, it supports the calculation and ranking of edge-cloud collaborative AI algorithm indicators
+
+## Proposal
+
+### Edge-Cloud Collaborative AI Dataset and Corresponding Data Processing Script
+
+It will provide an open-source edge-cloud collaborative AI dataset management module that is convenient for algorithm developers to use, which can be used as a plug-in for Sedna. The following functions will be supported:
+
+- Dataset management of edge-cloud collaborative AI
+- Quick download and use of datasets
+
+### Industrial Distributed Collaborative System Simulation
+
+It will provide Ianvs with docker in docker system emulation features.
+
+## Design Details
+
+### Dataset automatic download function
+
+The current `testenv.yaml` file is as follows
+
+```yaml
+testenv:
+  # dataset configuration
+  dataset:
+    # the url address of train dataset index; string type;
+    train_url: "/ianvs/dataset/train_data/index.txt"
+    # the url address of test dataset index; string type;
+    test_url: "/ianvs/dataset/test_data/index.txt"
+
+  # metrics configuration for test case's evaluation; list type;
+  metrics:
+      # metric name; string type;
+    - name: "f1_score"
+      # the url address of python file
+      url: "./examples/pcb-aoi/singletask_learning_bench/testenv/f1_score.py"
+```
+
+For the convenience of developers, the expected testenv.yaml file is as follows:
+
+- The user fills in the `dataset_url`, here is the HUAWEI CLOUD OBS link.
```

Comment:
Does dataset url only support HUAWEI CLOUD obs service?

Reply:
I think the common OBS links all can be used to download.